app/controllers/tokens_controller.rb

```diff
@@ -3,6 +3,8 @@ class TokensController < ApplicationController
   before_action :set_desired_path, only: [:show]
   before_action :ensure_token_auth_enabled!
 
+  rescue_from Token::MaxNumberOfTokensError, with: :throttle_limit_error
```

Comment:
Yeh, I know what you mean but I wanted to have something declarative around catching that particular error rather than wrapping the code in a rescue block.